src/store/store.js

```diff
@@ -25,8 +25,7 @@ const store = createStore(
   allReducers,
   compose(
     applyMiddleware(sagaMiddleware),
-    window.__REDUX_DEVTOOLS_EXTENSION__
```

Comment:
don't remove it:) just fix for users who don't have this extension

Reply:
This was by mistake.